internal/cli/up.go

```diff
@@ -104,8 +110,7 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 		}
 	}()
 
-	// TODO(maia): send along globalYamlManifest (returned by GetManifest...Yaml above)
-	err = upper.CreateManifests(ctx, manifests, globalYAML, c.watch)
+	err = upper.CreateManifests(ctx, manifests, globalYAML, c.watch, absTfPath)
```

Comment:
i'm a little bit worried about how this API is evolving, where you have different sets of arguments that are optional in different cases.

maybe we need to break this up into two separate methods?
upper.Start(tfPath, watchMounts)
upper.StartForTesting(manifests, yaml, watchMounts)
?

Reply:
Yeah I like moving the parsing of the tiltfile to only one place. I'll make a story for this.